instrumentation/lettuce/lettuce-4.0/javaagent/build.gradle.kts

```diff
@@ -20,5 +20,6 @@ dependencies {
 tasks.withType<Test>().configureEach {
   // TODO run tests both with and without experimental span attributes
   jvmArgs("-Dotel.instrumentation.lettuce.experimental-span-attributes=true")
+  jvmArgs("-Dotel.instrumentation.lettuce.connect-telemetry.enabled=true")
```

Comment:
The flag in netty instrumentation is a slightly different name `otel.instrumentation.netty.connection-telemetry.enabled`. Up to you to decide whether these should follow the same naming. I pushed a change to this PR that adds the flag to README.